src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/RefreshEmployerLevyData/RefreshEmployerLevyDataCommandHandler.cs

```diff
@@ -75,6 +81,23 @@ public class RefreshEmployerLevyDataCommandHandler : AsyncRequestHandler<Refresh
             }
         }
 
+        /// <summary> 
+        /// If there are any Submissions from Hmrc that have the same submission Id, we should discard all 
+        /// but the first. 
+        /// </summary> 
+        private DasDeclaration[] FilterDuplicateHmrcDeclarations(string empRef,
+            DasDeclaration[] declarations)
+        { 
+            var duplicateIds = declarations.GroupBy(d => d.SubmissionId).Where(g => g.Count() > 1)
+                .Select(s => s.First().SubmissionId).ToList(); 
+ 
+            duplicateIds.ForEach(submissionId => 
+                _logger.Info($"PayeScheme '{empRef}' has a duplicate submission id from Hmrc = '{submissionId}'")); 
+
+
+            return declarations.DistinctBy(x => x.SubmissionId).ToArray(); 
```

Comment:
This will take the first submission id from a group, whatever that happens to be. Is they're all have identical values that there is no difference, but what happens if they're different - is this still an acceptable way to select the row to retain?